lib/index.js

```diff
@@ -73,7 +73,10 @@ internals.schema = Joi.object({
     profileParams: Joi.object(),
     skipProfile: internals.flexBoolean.optional().default(false),
     forceHttps: internals.flexBoolean.optional().default(false),
-    location: Joi.string().optional().default(false),
+    location: Joi.alternatives().try(
```

Comment:
This does do the same thing as it did before in the case of not specifying a location. As in, without a location option, location will be undefined. I'll fix that since it's easy.